@xen-orchestra/xapi/src/index.js

```diff
@@ -32,8 +32,9 @@ const hasProps = o => {
 }
 
 class Xapi extends Base {
-  constructor({ maxUncoalescedVdis, ...opts }) {
+  constructor({ ignoreNobakVdis, maxUncoalescedVdis, ...opts }) {
```

Comment:
Either a default value or assert that it's defined :slightly_smiling_face: 

I should have done this with `maxUncoalescedVdis`, I'll do this in the future.